### ISSUE TITLE
Prevent Wincollect 10 Console installation

### ISSUE THREAD

**Issue by rka** [ibm-security-intelligence/wincollect#8]
Is there a way to prevent the console part to install? I tried to find the config for this but could not :(

**Comment by jamiewheaton** [comment]
In the GUI install, select advanced install and on the next screen, you can set the console to not be installed.  Currently not an option via cmd line, but something we are looking into.
![image](https://user-images.githubusercontent.com/15018160/135466590-f82188a1-d2d5-481e-825a-a7427554f794.png)

**Comment by rka** [comment]
Thanks a lot for the info, we will definitely wait for that feature. 
I know its a hard ask but is there an eta on this maybe?

**Comment by jamiewheaton** [comment]
I'm assuming the feature you would like is to set this option via the command line?  I can see this coming up soon in the future with customers not wanting the console installed.  I will create a story if this is the case and if you could give me a customer name to reference this helps with prioritization.

**Comment by rka** [comment]
Yes we would like to install the agent without the console basically. It would be nice if there was an install Parameter for it.
Like these: https://www.ibm.com/docs/en/qradar-common?topic=10-wincollect-command-line-installation-advanced-examples

**Comment by jamiewheaton** [comment]
I have created a story for this, what customer name can I add to it?

**Comment by rka** [comment]
Sorry but what is a "story"?  I am not sure I can share customer name here.

**Comment by jamiewheaton** [comment]
A story is an item to track work on a specific task, it's to track features in Agile.  You can send the customer information to jamie.wheaton@ca.ibm.com
